vcfa/remove_leftovers_test.go

```diff
@@ -118,6 +118,26 @@ func removeLeftovers(govcdClient *govcd.VCDClient, verbose bool) error {
 		}
 	}
 
+	// --------------------------------------------------------------
+	// Regions
```

Comment:
I think regions should be above vCenters to match cleanup hierarchy and avoid constraint errors

Reply:
Indeed, fixed